internal/resources/grafana/resource_sso_settings_test.go

```diff
@@ -63,6 +63,54 @@ func TestSSOSettings_basic(t *testing.T) {
 	}
 }
 
+func TestSSOSettings_basic_saml(t *testing.T) {
```

Comment:
Did you run this manually? v11.1 won't be released for a while

Reply:
Yes, I manually ran this test locally against the main branch. I'm not sure how to make it run in CI at the moment because it requires enterprise and some flags to be enabled.